auth/cache_test.go

```diff
@@ -13,189 +13,283 @@ import (
 	"golang.org/x/oauth2"
 )
 
-func TestWrite_DirectoryExistsFileExists(t *testing.T) {
+func TestRead(t *testing.T) {
+
 	require := requirepkg.New(t)
 	assert := assertpkg.New(t)
 
-	credentialDirectory, credentialPath, err := setup()
-	require.NoError(err)
-	require.NotNil(credentialDirectory)
+	testCases := []struct {
+		name          string
+		caseSetup     func(string, string) error
+		expectedError string
+	}{
+		{
+			name:          "No Directory, No File",
+			caseSetup:     func(dirPath, credPath string) error { return nil },
```

Comment:
No need to name these variables since they're not used
```suggestion
			caseSetup:     func(string, string) error { return nil },
```